examples/low_frequency/multiphysics/maxwell_icepak.py

```diff
@@ -53,35 +61,66 @@
 )
 # -
 
+# ### Units
```

Comment:
@Devin-Crawford @SMoraisAnsys @Samuelopez-ansys we should also agree on the heading levels.
If I'm not wrong, in all the other examples, each section starts with a heading level 2 (##), do we want to change it to 3 (###)?